data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -163,6 +164,12 @@ then
 	touch config-bootstrap.done
 fi
 
+# TODO: host-etcd endpoint needs to be rendered by cluster-etcd-operator
+if [ -z "$CLUSTER_ETCD_OPERATOR_IMAGE" ]
+then
+    sed -i '/192.0.2.1/I,+1 d' /opt/openshift/manifests/etcd-host-service-endpoints.yaml
```

Comment:
What is this magic address?

Reply:
This address is used during bootstrap to populate etcd hostnames [here](https://github.com/openshift/installer/blob/d3f582906205692828e09ec1c3e4f79de45a92f7/data/data/manifests/bootkube/etcd-host-service-endpoints.yaml.template#L11). The apiserver uses this endpoint to calculate etcd urls [here](https://github.com/openshift/cluster-kube-apiserver-operator/blob/a72999c2daba2f785541d7f92d3dddf4c073d8cb/pkg/operator/configobservation/etcd/observe_etcd.go#L66)

Our intention here is to make sure that `etcd-bootstrap` hostname is not present if CEO is disabled. We add it [here](https://github.com/openshift/installer/pull/2608/files#diff-36a138ec46315c3afef6bca683543ee8R173) but since we only get to know in bootkube.sh that CEO is disabled, we conditionally remove the hostname and ip address of the `etcd-bootstrap`